third_party/llvm/llvm.bzl

```diff
@@ -305,6 +305,13 @@ win32_cmake_vars = {
 # TODO(phawkins): use a better method to select the right host triple, rather
```

Comment:
This file was copied from TensorFlow. Would you mind investigating whether these changes should be made there too?

Reply:
I tried to compare & sync the two versions. I think the TF has everything, and I only ported over the required bits (TF has a number of extra configs which are not defined in TFRT)

Reply:
Thanks!